### ISSUE TITLE
Can't set two arrow keys as part of a shotcut

### ISSUE THREAD

**Issue by ColtonIdle** [rxhanson/Rectangle#312]
I want to have a shortcut for top left that is ctrl + option + left + up but I can only choose a single arrow key. this is something that moom  allows for example. Is this purposeful?

**Comment by rxhanson** [comment]
Thanks for the question.

This is because the shortcut recorder used in Rectangle, [MASShortcut](https://github.com/shpakovski/MASShortcut), only allows one non-modifier key to be used in a shortcut. I think this might be a limitation of the Carbon API in macOS that is used for keyboard shortcut registration. Moom might be listening to keystrokes in order to know when a shortcut is executed, as opposed to relying on the Carbon shortcut registration.

Long story short, I don't have plans to add capabilities to Rectangle beyond what's in MASShortcut. This keeps the app's code and behavior simple / easier to maintain. Closing this one out for now.

**Comment by ColtonIdle** [comment]
@rxhanson thanks for the super fast response. I'm unfamiliar with massShortcut but it makes sense that so many of these windowing utilities don't support what I want. They all likely use massShortcut under the hood. Going to open up an issue with them in the meantime.
